testData/testConfig.json

```diff
@@ -2,6 +2,12 @@
   "user-namespace": "jupyter-pods",
   "sub-dir": "/lw-workspace",
   "user-volume-size": "10Gi",
+  "gen3-license-user-maps-dynamodb-table": "gen3-license-user-maps",
```

Comment:
please update the config doc 🙏 https://github.com/uc-cdis/hatchery/blob/master/doc/howto/configuration.md